gravitee-apim-repository/gravitee-apim-repository-mongodb/src/main/java/io/gravitee/repository/mongodb/management/upgrade/upgrader/index/portalnavigationindex/PortalNavigationItemIndexUpgrader.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.repository.mongodb.management.upgrade.upgrader.index.portalnavigationindex;
+
+import io.gravitee.repository.mongodb.management.upgrade.upgrader.index.Index;
+import io.gravitee.repository.mongodb.management.upgrade.upgrader.index.IndexUpgrader;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PortalNavigationItemIndexUpgrader extends IndexUpgrader {
+
+    @Override
+    protected Index buildIndex() {
+        // Build a composite index on organizationId + environmentId
+        return Index.builder()
+            .collection("portal_navigation_items")
+            .key("organizationId", ascending())
+            .key("environmentId", ascending())
+            .name("organization_environment_idx")
```

Comment:
for the name of indices, we try to follow this convention:
`[a-z][1|-1]+`

the first letter of each key, followed by 1 or -1 depending on the sort order.
In you case it should be `o1e1`